app/search_builders/sufia_search_builder.rb

```diff
@@ -0,0 +1,9 @@
+class SufiaSearchBuilder < Blacklight::Solr::SearchBuilder
```

Comment:
On second thought let me see if it can combine.  I do not really want the AdvancedSearchBuilder, but it may do not harm